Tests/PointFreeTests/__Snapshots__/EpisodeTests/testEpisodeCredit_PublicEpisode_NonSubscriber_UsedCredit.1.Conn.txt

```diff
@@ -1879,8 +1879,8 @@ text, no markdown allowed. Here is some more text just to have some filler.</p>
               Subscribe to Point&#8209;Free
             </h3>
             <p class="body-leading fg-black normal lh-4 m-pb3 m-pt2">
-              You have access to this episode because you used a free episode credit. To get access to all past and
-future episodes, become a subscriber today!
+              This episode is free to all users. To get access to all past and future episodes, become a
```

Comment:
looks like this isn't using the correct mock data. this test is for when the user has used a credit.

Reply:
Good catches, thanks! Will look into it. Prob a prob with the mock updates.